File: Leetcode/392-Is-Subsequence/IsSubsequence.go
func isSubsequence(s string, t string) bool {
	ls := len(s)
	lt := len(t)
	if ls > lt {
		return false
	}
	i := 0
	j := 0
	for i < ls && j < lt {
		if s[i] == t[j] {
			i++
		}
		j++
	}
	return i == ls
}

File: Leetcode/392-Is-Subsequence/IsSubsequence.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isSubsequence = function(s, t) {
    if (s.length > t.length) {
        return false
    }
    var i=0
    var j=0
    while (i<s.length && j<t.length) {
        if (s[i] == t[j]) {
            i++
        }
        j++
    }
    return (i == s.length)
};
